.github/workflows/release.yml

```diff
@@ -98,9 +98,8 @@ jobs:
         with:
           package-dir: dist/${{ needs.build_sdist.outputs.SDIST_NAME }}
         env:
-          CIBW_BUILD: "cp310-* cp311-* cp312-* cp313-*"
           # Skip 32 bit builds and musllinux due to lack of numpy wheels
-          CIBW_SKIP: "*-win32 *_i686 *-musllinux*"
+          CIBW_SKIP: "*-win32 *_i686 *-musllinux* cp3*t-"
```

Comment:
Hmm, actually, this does nothing, because the freethreaded one is missing the trailing `*`.

Reply:
thank you. Fixed